Index.js

```diff
@@ -1,19 +1,21 @@
 /* eslint-disable no-console */
 const express = require('express')
 const {
-  createNewPerson, getAllContacts, getContactById, uploadCSVFile, uploadContacts
+  createNewPerson, getAllContacts, getContactByIdentifier, uploadCSVFile, sendTextMessage, uploadContacts, getContactByIdentifier
```

Comment:
For the future there is no need to replace `id` for `identifier`. They mean the same thing, but identifier is more verbose, and takes more space.